app/controllers/user_actions_controller.rb

```diff
@@ -2,13 +2,12 @@
 
 class UserActionsController < ApplicationController
   def index
-    params.require(:username)
-    params.permit(:filter, :offset, :acting_username, :limit)
+    user_actions_params.require(:username)
```

Comment:
Doing this in that order (using `#permit` first to remove what’s considered as bad data by Rails) allows us to raise if `username` was provided with bad data.